nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Customize styling

# Description
Customized styling to better match the MVC prototype. Changed primary and secondary colors, styled header, some individual pages. Also, disabled some unnecessary linter warnings.
Fixes # n/a
## Type of change
Please delete options that are not relevant.
# Testing Instructions
1. Checkout, check appsettings, and run.
2. Run ```npm i``` to install new dependencies, namely Node SASS
3. You'll see the new styling throughout the site.
# Checklist:
